app/javascript/article-form/components/Cover.jsx

```diff
@@ -0,0 +1,101 @@
+import { h, Component } from 'preact';
+import PropTypes from 'prop-types';
+import { Button } from '@crayons';
+import { generateMainImage } from '../actions';
+import { validateFileInputs } from '../../packs/validateFileInputs';
+
+export class Cover extends Component {
```

Comment:
The name of this class is not clear to me. Consider a more specific name, e.g. `<PostCoverImage />` or `<ArticleCoverImage />`

Reply:
Renamed to `ArticleCoverImage`.